homeassistant/helpers/json.py

```diff
@@ -20,6 +22,14 @@ def default(self, o: Any) -> Any:
             return o.isoformat()
         if isinstance(o, set):
             return list(o)
+        if isinstance(o, timedelta):
```

Comment:
We should not do this. This is not a common format for timedelta and this now applies to ALL of Home Assistant that uses this class.

What problem are you trying to solve ?